scripts/buildwithfdpp.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh
+
+GITDIR=../fdpp.git
```

Comment:
This should be overridable via cmd line
(and/or via env var)

Reply:
I thought you'd say that, but I figured as you have to copy the script into the top of the source tree editing it wouldn't be a problem. I'll add it.

Reply:
Maybe putting it to the top then?
Why would I want to copy it?

Reply:
Because git will remove any file under its control that didn't exist in the specific version to be bisected, hence the need to copy to a new path.